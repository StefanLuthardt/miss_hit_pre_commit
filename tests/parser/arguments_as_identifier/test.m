function test(x)
   arguments % keyword
      x uint
   end
   try
      arguments = 12; % identifier
   end
   arguments = 42; % keyword again not allowed in MISS_HIT
end
