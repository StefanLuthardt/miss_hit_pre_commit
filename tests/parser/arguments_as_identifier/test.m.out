In test.m, line 8
|    arguments = 42; % keyword again not allowed in MISS_HIT
|    ^^^^^^^^^ error: expected valid statement, found keyword 'arguments' instead
MISS_HIT Debug Summary: 1 file(s) analysed, 1 error(s)
